.travis.yml

```diff
@@ -12,9 +12,10 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
-before_script:
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
-  - "nvm install node 6.3.0"
+before_install:
+  - export CHROME_BIN=chromium-browser
```

Comment:
Yes, but only for the Jasmine/component tests. One of them fails in Firefox, so use Chromium there.
The e2e tests do not work with Chromium as the installed version in Travis is too old, so still use Firefox there.